src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1450,12 +1450,24 @@
   </Node>
 
   <Node Name="BoundSubPropertyPattern" Base="BoundNode">
-    <Field Name="Type" Type="TypeSymbol" Null="disallow"/>
-    <Field Name="Property" Type="Symbol" Null="allow"/>
-    <Field Name="ResultKind" Type="LookupResultKind"/>
+    <!-- Member is either a BoundPropertyPatternMember or an error expression -->
+    <Field Name="Member" Type="BoundExpression" Null="disallow"/>
     <Field Name="Pattern" Type="BoundPattern" Null="disallow"/>
   </Node>
 
+  <Node Name="BoundPropertyPatternMember" Base="BoundExpression">
+    <!-- Non-null type is required for this node kind -->
+    <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+
+    <Field Name="MemberSymbol" Type="Symbol" Null="allow"/>
+    <Field Name="Arguments" Type="ImmutableArray&lt;BoundExpression&gt;"/>
+    <Field Name="ArgumentNamesOpt" Type="ImmutableArray&lt;string&gt;" Null="allow"/>
+    <Field Name="ArgumentRefKindsOpt" Type="ImmutableArray&lt;RefKind&gt;" Null="allow"/>
+    <Field Name="Expanded" Type="bool"/>
+    <Field Name="ArgsToParamsOpt" Type="ImmutableArray&lt;int&gt;" Null="allow"/>
```

Comment:
The six properties above are not used right now and aren't handled properly by the pipeline (PreciseAbstractFlowPass, LocalRewriter, etc.), please remove them or comment them out. The reason for this ask is that, in case they ever come back, reviewers wouldn't be under impression that there is a meaningful implementation behind them.